### PR TITLE
Milestone 5: Fetching data

### PR DESCRIPTION
**Pull request by Lizdev-05**
In this milestone, I achieved the following...

- I used [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) for making API calls.
- I  fetched the book list from Bookstore API on the Books page load.
- I refactored add book and remove book features to persist your changes in the server.
- I made changes by using API.

### CODE REVIEW

File: src/redux/books/Books.js [hunk 1]
```diff
@@ -1,48 +1,60 @@
-import { v4 as uuidv4 } from 'uuid';
+// import { v4 as uuidv4 } from 'uuid';
```
Comment (BertrandConxy) [inline src/redux/books/Books.js]: - [ ] Uncommented codes make your codebase look bad and then it becomes difficult to read your codes. Please remove this commented line of code to make your code clean 😉 🏃

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @Lizdev-05 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 

## Highlights 💪 
- Project works fine
- good PR
- good README
- no linter errors


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Lizdev-05,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/Books.js (hunk 1 above)
Comment (Lizdev-05) [reply src/redux/books/Books.js]: Thank you for pointing that out. I will work on it.

**Comment by Lizdev-05** [comment]
Thanks for reviewing and approving my project, I appreciate it.

**Pull request merged**